src/HttpCache/EventListener/AddTagsListener.php

```diff
@@ -25,16 +27,21 @@
  * @see https://support.cloudflare.com/hc/en-us/articles/206596608-How-to-Purge-Cache-Using-Cache-Tags-Enterprise-only-
  *
  * @author Kévin Dunglas <dunglas@gmail.com>
+ * @author Yanick Witschi <yanick.witschi@terminal42.ch>
  *
  * @experimental
  */
 final class AddTagsListener
 {
     private $iriConverter;
+    private $purger;
+    private $debug;
 
-    public function __construct(IriConverterInterface $iriConverter)
+    public function __construct(IriConverterInterface $iriConverter, PurgerInterface $purger, bool $debug = false)
```

Comment:
Can you make the purger optional? It's useless when using the builtin one (it eases to use classes outside of Symfony / API Platform full stack).